content/applications/hr/fleet/service.rst

```diff
@@ -6,72 +6,282 @@ To properly maintain a fleet of vehicles, regular maintenance as well as periodi
+  edit...` to :ref:`add the service type and configure it <service/new-type>`.
+
+  .. note::
+     :guilabel:`Service Types` are **not** pre-configured in Odoo. When logging a service for the
+     first time, the *type* of service need to :ref:`first be created <service/new-type>` before it
+     can be selected.
+
+- :guilabel:`Date`: using the calendar popover, select the date the service was provided or is
+  scheduled to be performed. Navigate to the desired month using the :guilabel:`< > (arrow)` icons,
   then click on the date to select it.
 - :guilabel:`Cost`: enter the cost of the service.
-- :guilabel:`Vendor`: select the vendor who performed the service from the drop-down menu. If the
+- :guilabel:`Vendor`: select the vendor who performed the service using the drop-down menu. If the
   vendor has not already been entered in the system, type in the vendor name and click either
-  :guilabel:`Create` to add them, or :guilabel:`Create & Edit...` to create the vendor and edit the
-  vendor form. The vendor form allows for other details aside form the name to be entered, such as
-  contact information.
+  :guilabel:`Create` to add them, or  :guilabel:`Create and edit...` to :ref:`add and configure the
+  vendor <service/new-vendor>`.
 - :guilabel:`Vehicle`: select the vehicle that was serviced from the drop-down menu. When the
   vehicle is selected, the :guilabel:`Driver` field is populated, and the unit of measure for the
   :guilabel:`Mileage` field appears.
 - :guilabel:`Driver`: the current driver listed for the selected vehicle is populated when the
   :guilabel:`Vehicle` is selected. If the driver needs to be changed, another driver can be selected
   from the drop-down menu.
 - :guilabel:`Odometer Value`: enter the odometer reading when the service was done. The units of
-  measure will either be in kilometers (:guilabel:`km`) or miles (:guilabel:`mi`). When the
-  :guilabel:`Vehicle` is selected, the units of measure is populated. This comes from the vehicle
-  form. To change from kilometers to miles, or vice versa, click the :guilabel:`External Link`
-  button next to the vehicle selected in the :guilabel:`Vehicle` field. Change the unit of measure,
-  then click :guilabel:`Save`. the unit of measure will be updated in the :guilabel:`Odometer Value`
-  field.
+  measure will either be in kilometers (:guilabel:`km`) or miles (:guilabel:`mi`), depending on how
+  the selected vehicle was configured.
+
+  When the :guilabel:`Vehicle` is selected, the units of measure is populated. This comes from the
+  vehicle form.
+
+  To change from kilometers to miles, or vice versa, click the :guilabel:`Internal Link` button to
+  the right of the vehicle selected in the :guilabel:`Vehicle` field.
+
+  Change the unit of measure, then navigate back to the service form via the
+  :guilabel:`breadcrumbs`. The unit of measure is updated in the :guilabel:`Odometer Value` field.
```

Comment:
can possibly put this in a tip admonition, since changing the units of measure is not pertinent to filling out a service form

Reply:
Great idea!! I forget about "tips" and always use "notes". =)